src/main/resources/application.yml

```diff
@@ -20,6 +20,12 @@ mpesa:
 timer: "PT45S"
 
 server:
+  ssl:
+    key-alias: "tomcat-https"
+    key-store: "classpath:keystore.jks"
```

Comment:
can this be an external location maintained by DevOps?

Reply:
no idea.... need to read kubernetes doc.